src/main/java/ru/lanwen/wiremock/ext/WiremockFactory.java

```diff
@@ -14,9 +15,9 @@
  */
 class WiremockFactory {
 
-    public WireMockServer createServer(final Wiremock mockedServer) {
+    public WireMockServer createServer(final Wiremock mockedServer, ExtensionContext extensionContext) {
```

Comment:
can I ask you to keep the previous signature as well? To not break the binary compatibility. Thus you can avoid changes on the tests, adding nulls where the context should be

Reply:
Ok, and I've extracted the instantiation of the factory inside a private method. 

BTW, the method `newInstance()` is deprecated since Java 9: 
![image](https://user-images.githubusercontent.com/1970922/63089480-4bb9b600-bf58-11e9-80fd-11e91678b933.png)

Is it ok for you?

Reply:
yeah, j8 is the main target for now still, but thx for pointing